sway_libs/src/merkle_proof/binary_merkle_proof.sw

```diff
@@ -98,10 +98,10 @@ pub fn process_proof(
     key: u64,
     merkle_leaf: b256,
     num_leaves: u64,
-    proof: [b256; 2],
+    proof: [b256; 8],
```

Comment:
Why was this changed from 2->8 instead of 3->8?

Reply:
The tree I previously had in place wasn't a perfect binary Tree and only required 2 proofs.